src/containers/my-resources/lessons-container/LessonsContainer.tsx

```diff
@@ -49,28 +41,24 @@ const LessonsContainer = () => {
     columns,
     removeColumnRules
   )
+  const queryArgs = {
+    limit: itemsPerPage,
+    skip: (page - 1) * itemsPerPage,
+    sort,
+    title: searchTitle.current,
+    categories: selectedItems
+  }
 
-  const onResponseError = useCallback(
-    (error: ErrorResponse) => {
-      setAlert({
-        severity: snackbarVariants.error,
-        message: error ? `errors.${error.code}` : ''
-      })
-    },
-    [setAlert]
-  )
+  const {
+    data: response,
+    isLoading,
+    isSuccess,
+    refetch
+  } = useGetLessonsQuery(queryArgs)
 
-  const getMyLessons = useCallback(
-    () =>
-      ResourceService.getUsersLessons({
-        limit: itemsPerPage,
-        skip: (page - 1) * itemsPerPage,
-        sort,
-        title: searchTitle.current,
-        categories: selectedItems
-      }),
-    [page, itemsPerPage, sort, searchTitle, selectedItems]
-  )
+  const handleRefetch = () => {
```

Comment:
I guess creation of this function is redundant, just use void refetch() below

Reply:
this is for the purpose that for now some other components are still using fetchData value received from axios. To keep it working unless all components will use rtk, should i leave it?

Reply:
But, as I know, fetchData is an async function, so it returns the promise too, you can use `void fetchData `or `void refetch` for rtk query